work17/index.js

```diff
@@ -0,0 +1,191 @@
+
+    for (let i = 0; i < imagesDataArray.length; i++) {
+        const li = document.createElement("li");
+        const img = document.createElement("img");
+
+        img.src = imagesDataArray[i].src;
+        li.classList = "slider_items";
+        if (!i) {
+            li.classList.add("is-display");
+        }
+        li.id = `js-sliderItems${i + 1}`;
+
+        li.appendChild(img);
+        fragment.appendChild(li);
+    }
+
+    document.body.appendChild(wrapDiv)
+        .appendChild(slideDiv).appendChild(ul).appendChild(fragment);
+    slideDiv.appendChild(renderPagingNumber());
+    renderSlideArrow();
+    clickedSliderEvent();
+}
+
+const renderPagingNumber = () => {
+    let count = 1;
+    const sliderListElements = [...document.querySelectorAll(".slider_items")];
+
+    const span = document.createElement("span");
+    span.classList = "slider_pageNum";
+
+    sliderListElements.forEach(element => {
+        if (element.classList.contains("is-display")) {
+            span.textContent = `${count} / ${sliderListElements.length}`;
+            return span;
+        }
+    });
+    return span;
+};
+
+const renderSlideArrow = () => {
+    const wrapDiv = document.querySelector(".slider_slide");
+    const divLeft = document.createElement("div");
+    const divRight = document.createElement("div");
+    const buttonLeft = document.createElement("button");
+    const buttonRight = document.createElement("button");
+    const iLeft = document.createElement("i");
+    const iRight = document.createElement("i");
+
+    divLeft.classList = "slider_arrowBox-left slider_arrowBox";
+    divRight.classList = "slider_arrowBox-right slider_arrowBox";
+    buttonLeft.classList = "slider_prev slider_button"
+    buttonRight.classList = "slider_next slider_button"
+    buttonLeft.type = "button";
+    buttonLeft.disabled = true;
+    buttonRight.type = "button";
+    iLeft.classList = "fas fa-angle-left fa-3x";
+    iRight.classList = "fas fa-angle-right fa-3x";
+
+    divLeft.appendChild(buttonLeft).appendChild(iLeft);
+    divRight.appendChild(buttonRight).appendChild(iRight);
+
+    wrapDiv.appendChild(divLeft);
+    wrapDiv.appendChild(divRight);
+}
+
+const addOrRemoveDisabled = (index, nextOrPrevButton) => {
+    const sliderListElements = document.querySelector(".slider_item");
+    const targetElem = document.getElementById(`js-sliderItems${index}`);
+
+    const firstElem = sliderListElements.firstElementChild;
+    const lastElem = sliderListElements.lastElementChild;
+
+    if (targetElem === firstElem || targetElem === lastElem) {
+        nextOrPrevButton.disabled = true;
+    } else {
+        const disabledElem = document.querySelector("[disabled]");
+        disabledElem && disabledElem.removeAttribute("disabled");
+    }
+}
+
+const changePageNum = (currentImgNum, maxImgNum) => {
+    const pageNumElement = document.querySelector(".slider_pageNum");
+    pageNumElement.textContent = `${currentImgNum} / ${maxImgNum}`;
```

Comment:
IMO
The maximum number of sheets is added here as well,
Even if you move the slide, the number of sheets does not increase, so I thought that the maximum value is unnecessary.

Reply:
I'm sorry.
I did not understand what you pointed out, could you elaborate?